### PR TITLE
[REVIEW] Show install command for RMM

### PR DESCRIPTION
**Pull request by hcho3**
RMM version is now 0.14. Readme should show the updated command for installation.

### CODE REVIEW

**Comment by jakirkham** [comment]
@harrism, do you know how many versions of installs we should list here? Should any of these start to fall off (like RAPIDS 0.11)?

**Comment by kkraus14** [comment]
I'm in favor of just dropping the version entirely and just specifying a loose `rmm`

**Comment by harrism** [comment]
Agree with @kkraus14 -- the broadest solution that will require the least maintenance and work for the most people.

File: README.md [hunk 1]
```diff
@@ -20,6 +20,13 @@ For information on the interface RMM provides and how to use RMM in your C++ cod
 
 RMM can be installed with conda ([miniconda](https://conda.io/miniconda.html), or the full [Anaconda distribution](https://www.anaconda.com/download)) from the `rapidsai` channel:
 
+For `rmm version == 0.14` :
+```bash
+# for CUDA 10.2
+conda install -c nvidia -c rapidsai -c conda-forge -c defaults \
+    rmm=0.14 cudatoolkit=10.2
```
Comment (harrism) [inline README.md:27]:
```suggestion
    rmm cudatoolkit=10.2
```

**Comment by harrism** [comment]
@kkraus14 are you OK with the change this way?

**Pull request merged**